assets/scss/custom.scss

```diff
@@ -343,6 +343,17 @@ footer {
     }
 
     &.object-table, &.response-table {
+        border: 1px $table-border-color solid;
+
+        caption {
+            // the caption is outside the table's border box,
+            // so we have to give it its own border.
+            border: 1px $table-border-color solid;
+
+            // ... but avoid double border between caption and table
+            border-bottom: 0px;
```

Comment:
web developer extreme nit: it's `0`, not `0px` (unitless)

Reply:
```suggestion
            border-bottom: 0;
```